eZ/Publish/Core/FieldType/Url/UrlStorage/Gateway.php

```diff
@@ -61,6 +61,7 @@ abstract class Gateway extends StorageGateway
      *
      * @param int|string $fieldId
      * @param int $versionNo
+     * @param array|null $excludeUrlIds
      */
-    abstract public function unlinkUrl($fieldId, $versionNo);
+    abstract public function unlinkUrl($fieldId, $versionNo, $excludeUrlIds);
```

Comment:
optional, also making it null is pointless due to the usage
```suggestion
    abstract public function unlinkUrl($fieldId, $versionNo, array $excludeUrlIds = []);
```